src/Microsoft.TemplateEngine.Core/Conditional.cs

```diff
@@ -2,95 +2,191 @@
 using System.IO;
 using System.Text;
 using Microsoft.TemplateEngine.Abstractions.Engine;
+using System;
+using System.Runtime.CompilerServices;
```

Comment:
Makes sense

From: Mike Lorbetske [mailto:notifications@github.com]
Sent: Tuesday, August 23, 2016 11:09 AM
To: seancpeters/microsoft-templateengine <microsoft-templateengine@noreply.github.com>
Cc: Sean Peters <sepeters@microsoft.com>; Author <author@noreply.github.com>
Subject: Re: [seancpeters/microsoft-templateengine] Refactor (#1)


In src/Microsoft.TemplateEngine.Core/Conditional.cs<https://github.com/seancpeters/microsoft-templateengine/pull/1#discussion_r75920577>:

> @@ -2,95 +2,191 @@

>  using System.IO;

>  using System.Text;

>  using Microsoft.TemplateEngine.Abstractions.Engine;

> +using System;

> +using System.Runtime.CompilerServices;

Sort, system usings before others

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/seancpeters/microsoft-templateengine/pull/1/files/a425b2f9e55c8aa446463bd90060cb6bc99db3d0#r75920577>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AT8a8Uv1od7P3QEdbY1pkk1GFMMQyWkZks5qizc2gaJpZM4JqaeQ>.